contracts/governancestaking/BaseGovernanceShareField.sol

```diff
@@ -70,7 +71,8 @@ contract BaseGovernanceShareField is DAOContract {
 		
 				uint256 pending =
 					(userInfo.amount * accAmountPerShare) / 1e2
-					- userInfo.rewardDebt; // Divide 1e2 to reduce 2 Decimals since rewardDebt in 2 decimals so we can calculate how much reward earned in that cycle
+					- userInfo.rewardDebt; // Divide 1e2 to reduce 2 Decimals since rewardDebt in 18 decimals so we can calculate how much reward earned in that cycle
```

Comment:
why still 1e2?

Reply:
Because amount is GD so 2 decimals and accamountpershare 18 decimals so it makes 20 decimals but ill change accamountpershare possibly to 27 decimals after tests

Reply:
this explanation needs to be in the comment